demo/web/index.html

```diff
@@ -15,53 +16,47 @@
         document.getElementById("messages").appendChild(p);
       }
 
+      function wakeWordCallback(detection) {
+        writeMessage(
+                "Wake word detected: " + JSON.stringify(detection.label)
+        );
+      }
+      function inferenceCallback(inference) {
+        writeMessage(
+                "Inference detected: " + JSON.stringify(inference)
+        );
+      }
       async function startPicovoice(accessKey) {
         try {
           writeMessage("Picovoice is loading. Please wait...");
-          picovoiceWorker = await PicovoiceWebEnWorker.PicovoiceWorkerFactory.create(
-            {
-              accessKey: accessKey,
-              porcupineKeyword: { builtin: "Picovoice" },
-              rhinoContext: { base64: CLOCK_CONTEXT_64 },
-              start: true,
-            }
+          const porcupineModel = {
+            publicPath: "porcupine_params.pv",
+            forceWrite: true
+          }
+          const rhinoModel = {
+            publicPath: "rhino_params.pv",
+            forceWrite: true
+          }
+          let picovoiceWorker = await PicovoiceWeb.PicovoiceWorker.create(
+              accessKey,
+              PorcupineWeb.BuiltInKeyword.Picovoice,
```

Comment:
I understand why you use this builtin keyword here, but importing the `porcupine-web` package at first seems a bit weird, can we use a custom keyword?